### PR TITLE
issue/1514: allow handlebars to reference view functions directly

### PR DESCRIPTION
**Pull request by oliverfoster**
as per #1514

### CODE REVIEW

File: src/core/js/helpers.js [hunk 1]
```diff
@@ -92,6 +92,34 @@ define([
         },
 
         /**
+         * Allow templates to references view functions
```
Comment (tomgreenfield) [inline src/core/js/helpers.js]: Typo: `reference`.

File: src/core/js/helpers.js [hunk 2]
```diff
@@ -92,6 +92,34 @@ define([
...
+            var root = context.data.root;
+            var name = context.name;
+            var view = root.view;
+            
+            if (!view) {
+                Adapt.log.error('Missing helper: "'+name+'"', root);
+                return "";
+            }
+
+            switch (typeof view[name]) {
```
Comment (tomgreenfield) [inline src/core/js/helpers.js]: Possible to have a descriptive variable for `view[name]`?

**Comment by oliverfoster** [comment]
i've gone with:
```
{{{view.helper params}}}
```

**Comment by tomgreenfield** [comment]
So you didn't fancy going for an actual `view` helper?
```hbs
{{view <functionName> <params>}}}
```

Also, I'm getting a `Missing helper: "_classes"` warning for [Accordion](https://github.com/adaptlearning/adapt-contrib-accordion/blob/master/templates/accordion.hbs#L5) with this code.

**Comment by oliverfoster** [comment]
ok, now:
```handlebars
{{{view.calculatedSomething "value" 2}}}
{{{view.model.calculatedSomething "value" 2}}}
```

**Comment by oliverfoster** [comment]
this is ready for +1ing i think.

**Pull request merged**